### PR TITLE
feat: support comment after #[test] attribute

### PR DESCRIPTION
**Pull request by MunifTanjim**
Resolves https://github.com/rouge8/neotest-rust/issues/37

### CODE REVIEW

File: lua/neotest-rust/init.lua [hunk 1]
```diff
@@ -104,7 +104,7 @@ local function integration_test_name(path)
 end
 
 function adapter.discover_positions(path)
-    local query = [[
+    local query = [[;; query
```
Comment (rouge8) [inline lua/neotest-rust/init.lua:107]: what does this change do?

**Review by rouge8 — CHANGES_REQUESTED** [review]
can you add a test?

File: lua/neotest-rust/init.lua (hunk 1 above)
Comment (MunifTanjim) [reply lua/neotest-rust/init.lua:107]: That's just for treesitter injection to highlight it as `query`:

https://github.com/nvim-treesitter/nvim-treesitter/blob/ec97c5ec720da7b8bb5f53572c91c5b144ae0cae/queries/lua/injections.scm#L26-L27

I can remove that if you want.

Comment (rouge8) [reply lua/neotest-rust/init.lua:107]: oh that's cool, you can leave it in

File: tests/data/simple-package/src/lib.rs [hunk 2]
```diff
@@ -4,4 +4,10 @@ mod tests {
     fn math() {
         assert_eq!(1 + 1, 2);
     }
+
+    #[test]
+    /// same string
+    fn same_string() {
+        assert_eq!("robot".to_string(), String::from("robot"));
+    }
```
Comment (MunifTanjim) [inline tests/data/simple-package/src/lib.rs:12]: Added it here. Lemme know if you want it someplace else instead.

@rouge8

**Review by rouge8 — APPROVED** [review]
Thanks!

**Pull request merged**